pyrogram/scaffold.py

```diff
@@ -41,7 +41,7 @@ class Scaffold:
 
     PARENT_DIR = Path(sys.argv[0]).parent
 
-    INVITE_LINK_RE = re.compile(r"^(?:https?://)?(?:www\.)?(?:t(?:elegram)?\.(?:org|me|dog)/joinchat/)([\w-]+)$")
+    INVITE_LINK_RE = re.compile(r"^(?:https?://)?(?:www\.)?(?:t(?:elegram)?\.(?:org|me|dog|one)/joinchat/)([\w-]+)$")
```

Comment:
Yes should be (?:elegram|elega) or (?:elegr?am?)

Reply:
It better be:
`    INVITE_LINK_RE = re.compile(r"^(?:https?://)?(?:www\.)?(?:(?:t(?:elegram)?\.(?:org|me|dog)|telega\.one)/joinchat/)([\w-]+)$")`.

Since with the `.one` domain, there is only `telega.one` and not the others like the other domains.
For example:
- `t.one`
- `telegram.one`
- `telega.dog`

`telega.org` and` telega.me` exist but do not belong to Telegram (I think).